Package.swift

```diff
@@ -1,4 +1,4 @@
-// swift-tools-version:5.1
+// swift-tools-version:5.5
```

Comment:
requires Swift 5.5

Reply:
So Xcode 13 only?

Reply:
Yup, for now there were other reasons why we needed Xcode 13 (I think async support being one), so this is 🆗

Reply:
storekit 2 requires async / await, which in turn requires xcode 13. 
Also, we started offering async / await alternatives for methods with iOS 13 requirements, and doing so requires Xcode 13.2 specifically, since previous iterations didn't have support for swift concurrency on iOS 13